src/main/java/io/anserini/search/topicreader/TopicReader.java

```diff
@@ -41,6 +44,9 @@
 public abstract class TopicReader<K> {
   protected final int BUFFER_SIZE = 1 << 16; // 64K
   protected Path topicFile;
+  final private static String CACHE_DIR = System.getProperty("user.home") + "/.cache/anserini/topics-and-qrels";
```

Comment:
yea... not sure how portable this is for Windows?
but not sure we need to worry about it at this time?
more just a note?

Reply:
Right. I think I'll use some Path operator so the jre will figure it out on different platforms then.